### ISSUE TITLE
[F.Req] Baby mode

### ISSUE THREAD

**Issue by yum13241** [juanmuscaria/a-mod-made-to-corrupt-a-well-known-block-game#3]
It would prevent this "mod" from corrupting things such as the classpath, calls to malloc(), or anything that would prevent the game from starting, so it would at least reach the title screen. Hotkeys to trigger the JVM corruption would be nice, in case you're using a seed that causes the game to crash when you press ESC.

Or better yet, don't start the corruption until the title screen is loaded or a world is loaded, based on a JVM arg or smth. There should probably be a warning that they might miss out on cool corruptions.

Also, will this mod work on non-Forge loaders?

**Comment by someaccidcabout** [comment]
yeah i think it having not start until tile screen be nice bc i cant pass loading screen at all.
